_posts/en/newsletters/2022-07-27-newsletter.md

```diff
@@ -17,7 +17,7 @@ to popular Bitcoin infrastructure software.
 
 ## News
 
-- **Multiformat single-sig message signing:** Bitcoin Core and many
+- **Multiformat single-sig message signing:** {% include functions/podcast-callout.md url="pod210 message signing" %} Bitcoin Core and many
```

Comment:
I don't have a strong opinion, but what do y'all think about maybe putting the headphones icon at the end of an item?  E.g., for this item, it would be:

> [...]
>
> Sherief is planning to write a BIP specifying the behavior.  [ICON]

I think that might look a bit better and would seem to fit better in my mind, where I'd expect people to read the item first and then look at the commentary.

Alternatively, maybe we can rework the permalink bullets to be a two-column layout, so that items without an audio discussion would look like:

> [PERM] [NULL] **Multiformat single-sig message signing:** [...]

And items with a audio discussion would be:

> [PERM] [AUDIO] **Multiformat single-sig message signing:** [...]

I think that could be done by changing `auto-anchor.rb` to handle the `[PERM] [NULL]` case, and then when @bitschmidty or whoever comes back to add the audio links, they would change the source bullet from a `-` to a `*` with the helper specifying the older anchor link, e.g.:

```diff
--- foo	2023-01-25 14:07:45.184243414 -1000
+++ bar	2023-01-25 14:09:33.032093620 -1000
@@ -1 +1 @@
-- **Multiformat single-sig message signing:** Bitcoin Core and many
+* {% include functions/podcast-callout.md anchor="multiformat-single-sig-message-signing" url="pod210 message signing" %} **Multiformat single-sig message signing:** Bitcoin Core and many
```

(We could probably combine those two parameters into one by having the podcast page anchors use the same anchors as the main newsletter.  That might also allow us to avoid having to add the extra link definitions to the bottom of each page.)

Reply:
I do like the idea of putting it at the end of the item as you suggest. It has a natural feel, and it also makes it clearer that the headphones are not part of the original item’s title which often is a link in itself.

Reply:
Yeah, we could then make it a style rule not to end any item description with a link.  Putting the icon at the end would also work for the year-in-review edition where each free-form paragraph is its own item even though there are no bullets.  And it might also work with putting the icon at the end of section headers that you discuss as a unit (e.g. contributed blog posts that we include in the newsletter), e.g.

> ### Foo heading [ICON]